api/src/test/java/org/openmrs/module/dhisreport/api/adx/AdxTypeTest.java

```diff
@@ -0,0 +1,107 @@
+package org.openmrs.module.dhisreport.api.adx;
+
+import static org.junit.Assert.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.io.UnsupportedEncodingException;
+import java.io.Writer;
+import java.math.BigDecimal;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Map;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+import javax.xml.namespace.QName;
+import javax.xml.xpath.XPathExpressionException;
+import org.junit.Test;
+
+public class AdxTypeTest {
+
+	@Test
+	public void ShouldHaveProperValuesInAllAttributesOfAdxType() throws JAXBException, FileNotFoundException,
```

Comment:
Can you change the method name to match these conventions? http://www.oracle.com/technetwork/java/codeconventions-135099.html